x-pack/plugins/fleet/public/components/enrollment_instructions/manual/index.tsx

```diff
@@ -64,7 +70,9 @@ sudo elastic-agent enroll ${enrollArgs} \nsudo systemctl enable elastic-agent \n
 sudo rpm -vi elastic-agent-${agentVersion}-x86_64.rpm
 sudo elastic-agent enroll ${enrollArgs} \nsudo systemctl enable elastic-agent \nsudo systemctl start elastic-agent`;
 
-  const googleCloudShellCommand = `gcloud config set project <PROJECT_ID> && \nFLEET_URL=${fleetServerUrl} ENROLLMENT_TOKEN=${enrollmentToken} STACK_VERSION=${agentVersion} ./deploy.sh`;
+  const googleCloudShellCommand = `gcloud config set project ${gcpProjectId} && ${
+    gcpAccountType === 'organization-account' ? `\nORG_ID=${gcpOrganizationId}` : ``
+  } \nFLEET_URL=${fleetServerUrl} ENROLLMENT_TOKEN=${enrollmentToken} STACK_VERSION=${agentVersion} ./deploy.sh`;
```

Comment:
there's a weird extra new line for Organization:

![image](https://github.com/elastic/kibana/assets/19270322/96d5a8d2-a3b7-4f5b-acfa-c627702380f0)

I think we can change it to:

```suggestion
  const googleCloudShellCommand = `gcloud config set project ${gcpProjectId} && \n${
    gcpAccountType === 'organization-account' ? `ORG_ID=${gcpOrganizationId} ` : ''
  }FLEET_URL=${fleetServerUrl} ENROLLMENT_TOKEN=${enrollmentToken} STACK_VERSION=${agentVersion} ./deploy.sh`;
```